.changeset/wild-pandas-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Add support for Queue Consumer concurrency
```

Comment:
This should have some more detail around the the user facing changes (new config options?)

Reply:
✅ Updated